### DISCUSSION TITLE
How to deal with Jsonb type and json filter

### DISCUSSION THREAD

**Discussion by bala-ravuthan** [hasura/graphql-engine#8150] [Question]
trying to connect hasura graphql, spring boot, postgres and filtering the result from db, the result is in jsonb type. want to apply filters on the json result. please let know how to apply filters on jsonb type result.

**Comment by bala-ravuthan** [comment]
How to achieve the below scenario using spring boot with graphql and postgres:

https://user-images.githubusercontent.com/42561199/152734660-a4200dd7-60e3-4155-a372-421904e13153.png

**Comment by wmbutler** [comment]
I'm trying to do the same. I have psql fields that are `type:json` and I'd like to understand if hasura has support for this.

**Reply by arjunyel** [reply]
@wmbutler hi friend, do these docs help? https://hasura.io/docs/latest/queries/postgres/query-filters/#jsonb-operators-_contains-_has_key-etc
